src/content/docs/developer-tools/sdks/backend/nextjs-sdk.mdx

```diff
@@ -1704,6 +1704,48 @@ To check your configuration, the SDK exposes an endpoint with your settings.
 }
 ```
 
+## State not found error
+
+### Solution
+
+1. Confirm that the domain you start the auth flow from is different from the domain you are redirected to after the auth flow is complete. If this is not the case, see the explanation.
+2. Dynamically set the KINDE_SITE_URL and `KINDE_POST_LOGIN_REDIRECT_URL` when working with vercel preview domains.
+   If you are using Vercel, you can set the `KINDE_SITE_URL` and `KINDE_POST_LOGIN_REDIRECT_URL` dynamically.
+
+```jsx
```

Comment:
@peterphanouvong Can you add `title="next.config.js"` next to `jsx` here so the file name appears in a frame in the code block?